src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -74,7 +74,8 @@ export const DefaultTypedMessageImageRenderer = React.memo(function DefaultTyped
     return renderWithMetadata(
         props,
         <Typography variant="body1" style={{ lineBreak: 'anywhere' }} data-testid="text_payload">
-            <RenderText text={props.message.content}></RenderText>
+            {/* TODO: support different image type */}
+            <img style={{ width: '100%' }} src={`data:image/png;base64,${props.message.content}`} />
```

Comment:
I suggest using canvas to draw the image.

Reply:
Can you talk more details?

Reply:
data URL might be blocked by CSP rules. With CSP report mode, the picture even will be reported to the website leads to info leak.

Reply:
OK, I'm going to do it in the shuffling image PR.